wand/image.py

```diff
@@ -1250,7 +1250,80 @@ def resize(self, width=None, height=None, filter='undefined', blur=1):
             library.MagickSetSize(self.wand, width, height)
             if not r:
                 self.raise_exception()
+                
+    @manipulative
+    def resample(self, x_res=None, y_res=None, filter='undefined', blur=1):
+        """adjust the number of pixels in an image so that when displayed at the
+           given Resolution or Density the image will still look the same size in
+           real world terms.
```

Comment:
Sorry for late response.  Although it’s trivial, we haven’t aligned paragraphs’ left edges to leading text of content e.g.:

```python
"""blah
   blah
```

but leading quote character e.g.:

```python
"""Blah
blah

```

Also the leading character of every sentence should be uppercase.